post_office/mail.py

```diff
@@ -270,7 +272,26 @@ def send(email):
     Email.objects.filter(id__in=email_ids).update(status=STATUS.sent)
 
     email_ids = [email.id for (email, e) in failed_emails]
-    Email.objects.filter(id__in=email_ids).update(status=STATUS.failed)
+
+    numbers_mails_get_back_in_queue = 0
+    # Retry is enabled
+    if get_max_retry() > 0:
```

Comment:
`if get_max_retries():` to make it more pythonic.

Reply:
ok